### PR TITLE
HUE-5435 Allow dbproxy classpath to be configurable

### PR DESCRIPTION
**Pull request by z-york**

### CODE REVIEW

File: desktop/libs/librdbms/java/bin/dbproxy [hunk 1]
```diff
@@ -20,7 +20,11 @@ set -e
 
 export DBPROXY_HOME=$(cd $(dirname $0)/../.. && pwd)
 
-export CLASSPATH=$DBPROXY_HOME/java-lib/dbproxy-1.0.jar:$CLASSPATH
```
Comment (romainr) [inline desktop/libs/librdbms/java/bin/dbproxy]: Avoid the duplication?

e.g.

export CLASSPATH=$DBPROXY_HOME/java-lib/dbproxy-1.0.jar:$CLASSPATH

if [ -n "$1" ]; then
  export CLASSPATH="$1:$CLASSPATH"
fi

?

File: desktop/libs/notebook/src/notebook/management/commands/dbproxy_server.py [hunk 2]
```diff
@@ -36,7 +37,10 @@ def handle(self, *args, **kwargs):
 
     cmd = os.path.join(os.path.dirname(__file__), "..", "..", "..", "..", "..", "librdbms", "java", "bin", "dbproxy")
 
-    LOG.info("Executing %r (%r) (%r)" % (args, args, env))
+    command = ['dbproxy']
+    command.append(DBPROXY_EXTRA_CLASSPATH.get())
```
Comment (romainr) [inline desktop/libs/notebook/src/notebook/management/commands/dbproxy_server.py]: Actually, why not just?
if DBPROXY_EXTRA_CLASSPATH.get(): 
  env['CLASSPATH'] = '%s:%s' (DBPROXY_EXTRA_CLASSPATH.get(), env.get('CLASSPATH', '')
?

Comment (z-york) [inline desktop/libs/notebook/src/notebook/management/commands/dbproxy_server.py]: That would be much simpler. Let me test it out and upload an updated PR

**Comment by z-york** [comment]
Updated the commit.

**Comment by romainr** [comment]
Thanks!

**Pull request merged**